Use Cases/SDPi/Temp from Repo/tf1-ch-c-use-cases.adoc

```diff
@@ -0,0 +1,88 @@
+= Appendix C – Device Point-of-care Interoperability (DPI) Use Cases
+
+
+NOTE:  This file integrates the use cases that are linked to the SDPi profiles to provide functional requirements, including any introductory sections for how use cases are managed in SDPi. +
+{empty} +
+Note that each use case is in its own file. +
+{empty} +
+Note that AsciiDoc Gherkin (plugin?) may be used ... eventually ... !
+
+[appendix#vol1_appendix_c_dpi_use_cases,sdpi_offset=C]
+== Appendix C - Device Point-of-care Interoperability (DPI) Use Cases
+
+This Appendix describes a number of use cases that explore the various SDPi functions.
+
+This Appendix describes a number of use cases that explore the various SDPi functions.
+Each Use Case (also called a Feature in Gherkin) is organized as follows:
+
+•	*Narrative* – a description of the desired functionality from the user perspective
+•	*Technical Pre-Conditions* – any assumptions or pre-conditions
+•	*Scenario(s)* – the scenarios explore both the Happy Path when everything goes according to plan and the Alternative Paths where things do not go according to plan.
+
+##TODO:  Include model graphic showing relationships ...
+##
+// C.1
+[sdpi_offset=1]
+=== General Overview of DPI Use Cases & Analysis
+
+// C.1.1
+==== Rich History of Medical Device Interoperability (MDI) Use Cases
+
+The vision of plug-and-play medical device interoperability has been an active pursuit since the early 1980's, with the IEEE 1073 group's formation in those early years.  Over the 40+ years, many projects -- both industry and standards-based -- have contributed to an ever growing set of real-world use cases, and the ISO/IEEE 11073 SDC program is no different.
+
+Looking to leverage this wealth of use cases in considering SDPi, a "compendium of medical device oriented use cases" was created to facilitate referencing and use.  The use cases detailed in this appendix build on those that are captured in this document:  <<ihe_pcd_sdpi_use_cases_compendium_2019>>
+
+// C.1.2
+==== Overview of Architectural & Business Systems Concepts
+
+#TODO:  Include difference between logical actors and business (real world implementation) systems#
+
+// C.1.3
+==== Overview of DIS & DAS Medical Device Alert Distribution Systems
+
+##TODO:  Include Word content now or push to later (1.x) version?##
+
+// C.1.4
+==== Use Case Specification Conventions Using Cucumber/Gherkin
+
+<<<<<<< HEAD
+Each Use Case (also called a Feature in Gherkin) is organized as follows:
+
+•	*Narrative* – a description of the desired functionality from the user perspective
```

Comment:
No official list type in AsciiDoc. Please use`*`

Reply:
Updated